### ISSUE TITLE
Clear yarn cache on Travis (of incompatible sub-dependency)

### ISSUE THREAD

**Issue by cyk** [Addepar/ember-table#656]
This build is [failing master](https://travis-ci.org/Addepar/ember-table/builds/487241773?utm_source=github_status&utm_medium=notification) and recent PRs (https://github.com/Addepar/ember-table/pull/652, https://github.com/Addepar/ember-table/pull/649, https://github.com/Addepar/ember-table/pull/648). This is caused by a sub-dependency specifying node compatibility >= 8:

```sh
error ip-regex@3.0.0: The engine "node" is incompatible with this module. Expected version ">=8". Got "6.16.0"
error Found incompatible module
```

The `ip-regex` is a sub-dependency of ember-cli-addon-docs. They ran into the same issue. The workaround was to [pin the related dependency](https://github.com/ember-learn/ember-cli-addon-docs/pull/302) to an earlier version that supports Node 6.

Though ember-table's version constraint for ember-cli-addon-docs includes the fixed version, it appears that Travis's yarn cache continues to have the incompatible version (i.e., `ip-regex@3.0.0`).

**TL;DR:** Clearing the yarn cache on Travis should resolve the issue.

**Comment by Frozenfire92** [comment]
Is this best done via `.travis.yml` by disabling the cache?

https://github.com/Addepar/ember-table/blob/821270bdd1345e51c41de33e3f117199b0d11dca/.travis.yml#L14-L15

or via Travis interface for each build?

https://docs.travis-ci.com/user/caching/#clearing-caches

![](https://docs.travis-ci.com/images/caches-item.png)

Would love to get this going as #648 keeps popping up

**Comment by cyk** [comment]
@Frozenfire92 Good question. I think  removing the yarn cache altogether is the better option. The reduced build time isn't worth (potentially) inconsistent builds.

**Comment by bantic** [comment]
Recent builds at Travis seem to be passing without much issue, so I will go ahead and close this, but if I'm mistaken please feel free to reopen.
